app/views/partners/AdditionalPartnersRequired.scala.html

```diff
@@ -0,0 +1,51 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import views.html.components._
+@import views.html.layouts.layout
+
+@this(
+        layout: layout,
+        h1: h1,
+        p: p,
+        link: link,
+        button: button,
+        form: FormWithCSRF
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@layout(pageTitle = Some(titleNoForm(messages("partner.additional.required.heading"))), backLink = true) {
+
+    @h1(messages("partner.additional.required.heading"))
+
+    @p{
+        @messages("partner.additional.required.para.1")
+        @link(
+            link = appConfig.vat2Link,
+            messageKey = messages("partner.additional.required.para.link"),
+            isExternal = true,
+            isButton = false
+        )
+        @messages("partner.additional.required.para.2")
+    }
+
+    @form(controllers.routes.TaskListController.show) {
```

Comment:
Doesn't really need to be a form, we have the link component that has a toggle to make it pretend to be a button.